__tests__/lib/eval-queue.test.ts

```
@@ -0,0 +1,192 @@
+        async () => ({ ok: true }),
+        { priority: Priority.LOW },
+      );
+
+      const status = getQueueStatus();
+      expect(status.pending).toHaveLength(1);
+      expect(status.pending[0].itemName).toBe("queued-item");
+      expect(status.pending[0].priorityLabel).toBe("LOW");
+      expect(status.processing).toHaveLength(3);
+
+      // Clean up
+      blockers.forEach(r => r());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)